tool/tsh/common/hardware_key_test.go

```diff
@@ -0,0 +1,173 @@
+//go:build pivtest
+
+/*
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2024  Gravitational, Inc.
```

Year is getting closer  ;)